### PR TITLE
Create npx-portfolio.md

### PR DESCRIPTION
**Pull request by dispherical**
Thanks for making a PR! Please fill in the below:

Name: David (aboutdavid)

Do you run a club? Yes!

What relevant experiences do you have to help you write this Jam? I've made a similar portfolio before (`npx davidfolio`, i think)

Will you be available on Slack to provide support to Hack Clubbers who run into issues while going through your Jam? Yes

### CODE REVIEW

**Review by zeyuyaoy — CHANGES_REQUESTED** [review]
This looks super cool! I wanted to do something like this a while ago :)

Could you
1. answer the questions at the bottom from [the example file](https://github.com/hackclub/jams/blob/main/jams/example.md) and
2. include that Replit can be used to build this project, and add steps for those who are using Replit to set up this project?

**Review by sarthaktexas — CHANGES_REQUESTED** [review]
I would approve this as I'm very excited but you are missing the questions at the bottom. Please add them so this can be merged asap!

**Comment by dispherical** [comment]
Sure! I'll go ahead and add those things

**Comment by dispherical** [comment]
okay i went and added the questions + added the fact it could work on repl

File: jams/npx-portfolio.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+_Software used:_ npm and Node.JS
+
+_Customization opportunity:_ You'll be able to make your own portfolio that runs in the terminal, being able to customize it using colours and display images in the terminal! You'll be able to showcase your skills, learn basic operations
+
+_Platform limitations:_ This can only run in the terminal, but should support all operating systems since it uses Node.JS! Repl can be used to create this project
+
+_Time estimation:_ 1 hour
+
+### Breakdown
```
Comment (Arpan-206) [inline jams/npx-portfolio.md:17]: Can you add timestamps for each step in the breakdown?

**Review by Arpan-206 — CHANGES_REQUESTED** [review]
Time estimation thingies, looks good otherwise!

**Comment by sahitid** [comment]
sorry, although we loved this idea, we'll have to close this proposal due to the fact that you've missed the deadline.

**Pull request closed without merging**